spec/models/ext_management_system_spec.rb

```diff
@@ -312,6 +312,38 @@
     end
   end
 
+  describe "refresh" do
+    let(:ems) { FactoryBot.create(:ext_management_system) }
+
+    it "raises an error if the authentication check fails" do
+      allow(ems).to receive(:missing_credentials?).and_return(false)
+      allow(ems).to receive(:authentication_status_ok?).and_return(false)
+
+      expect { ems.refresh }.to raise_error(RuntimeError, "Provider failed last authentication check")
+    end
+
+    it "raises an error if no provider credentials are defined" do
+      allow(ems).to receive(:authentication_status_ok?).and_return(true)
+      allow(ems).to receive(:missing_credentials?).and_return(true)
+
+      expect { ems.refresh }.to raise_error(RuntimeError, "no Provider credentials defined")
+    end
+
+    it "calls the EmsRefresh.refresh method internally" do
+      allow(ems).to receive(:missing_credentials?).and_return(false)
+      allow(ems).to receive(:authentication_status_ok?).and_return(true)
+      allow(EmsRefresh).to receive(:refresh)
+
+      ems.refresh
+
+      expect(EmsRefresh).to have_received(:refresh)
+    end
+
+    it "defines a queue_refresh alias for refresh_ems" do
+      expect(ems.method(:refresh_ems)).to eql(ems.method(:queue_refresh))
```

Comment:
Just testing an alias isn't too helpful, think we should remove this test and follow-up with a test that actually tests the refresh_ems method

Reply:
Ok, removed.